model/implementation/TestSessionHistoryService.php

```diff
@@ -74,4 +75,17 @@ public function getBackUrl($delivery = null)
         }
         return parent::getBackUrl($delivery);
     }
+
+    /**
+     * @param $userId
+     * @return string
+     */
+    protected function getActor($userId)
+    {
+        /** @var LtiUserHelper $ltiUserHelper */
+        $ltiUserHelper = $this->getServiceLocator()->get(LtiUserHelper::SERVICE_ID);
```

Comment:
Instead, I prefer to use LtiUserService, otherwise, we will have too many abstractions